app/components/op_primer/border_box_table_component.sass

```diff
@@ -17,10 +17,17 @@
 
     &--row-item
       &:not(:first-child)
-        margin-left: 6px
+        padding-left: 6px
 
       &:not(:last-child)
-        margin-right: 6px
+        padding-right: 6px
+
+    &--heading
+      &:not(:first-child)
+        padding-left: 6px
+
+      &:not(:last-child)
+        padding-right: 6px
```

Comment:
Please combine that, so that we do not have to repeat the code. If we decide the change spacing somewhen later on, the chances to forget about one or the other is lower.

```
&--heading,
&--row-item
  &:not(:first-child)
    .....
```